guides/common/modules/ref_dhcp-isc-settings.adoc

```diff
@@ -0,0 +1,43 @@
+[id="DHCP_ISC_Settings_{context}"]
+= dhcp_isc Settings
+
+The _dhcp_isc_ provider uses a combination of the ISC DHCP server OMAPI management interface and parsing of configuration and lease files.
+This requires it to be run either on the same host as the DHCP server or to have network filesystem access to these files.
+You can define the following settings in `dhcp_isc.yml`:
```

Comment:
Rather than "you can define", should this be more generic?
```suggestion
The following settings are defined in `dhcp_isc.yml`:
```

My reasoning would be that the entire file is managed by the installer and in the manual the user is never supposed to touch these. That's also the reason why I'm very hesitant to include this in the first place. Documenting them gives the impression that it's supported to modify them, which I'd say is unsupported.

Reply:
Applied.

I was hoping to include this as a reference to let users know what could potentially be modified (using the installer).

Reply:
I get that, and I do think it's a good idea to document at least somewhere. I'm questioning what the right place is though. There are a lot of examples config files (https://github.com/theforeman/smart-proxy/tree/develop/config/settings.d). Perhaps we should enhance those to have comments. The packaging simply takes these (and from plugins) and ships them by default. The installer then has its own templates, but we can base them on that.

Or we can look at creating a new reference guide altogether. I'd love to have a guide "What is the Smart Proxy" that starts with a high level overview for users/contributors who want to **understand** the concepts (rather than simply use it). We could also use those for onboarding new people.

Food for thought.